q2_longitudinal/plugin_setup.py

```diff
@@ -28,10 +28,14 @@
 )
 
 
-base_parameters = {
+basest_parameters = {
```

Comment:
Yowzah! How about `shared_parameters`?

Reply:
😞 I guess I was just channeling a little shakespeare or something... "shared" it is!